airflow/gcp/example_dags/example_cloud_memorystore.py

```diff
@@ -0,0 +1,188 @@
+    )
+    # [END howto_operator_list_instances]
+
+    # [START howto_operator_list_instances_result]
+    list_instances_result = BashOperator(
+        task_id="list-instance-result", bash_command="echo \"{{ task_instance.xcom_pull('get-instance') }}\""
+    )
+    # [END howto_operator_list_instances_result]
+
+    # [START howto_operator_update_instance]
+    update_instance = CloudMemorystoreUpdateInstanceOperator(
+        task_id="update-instance",
+        location="europe-north1",
+        instance_id=INSTANCE_NAME,
+        project_id=GCP_PROJECT_ID,
+        update_mask={"paths": ["memory_size_gb"]},
+        instance={"memory_size_gb": 2},
+    )
+    # [END howto_operator_update_instance]
+
+    # [START howto_operator_set_acl_permission]
+    set_acl_permission = GoogleCloudStorageBucketCreateAclEntryOperator(
+        task_id="gcs-set-acl-permission",
+        bucket=BUCKET_NAME,
+        entity="user-{{ task_instance.xcom_pull('get-instance')['persistenceIamIdentity']"
+               ".split(':', 2)[1] }}",
+        role="OWNER",
+    )
+    # [END howto_operator_set_acl_permission]
+
+    # [START howto_operator_export_instance]
+    export_instance = CloudMemorystoreExportInstanceOperator(
+        task_id="export-instance",
+        location="europe-north1",
+        instance=INSTANCE_NAME,
+        output_config={"gcs_destination": {"uri": EXPORT_GCS_URL}},
+        project_id=GCP_PROJECT_ID,
+    )
+    # [END howto_operator_export_instance]
+
+    # [START howto_operator_import_instance]
+    import_instance = CloudMemorystoreImportInstanceOperator(
+        task_id="import-instance",
+        location="europe-north1",
+        instance=INSTANCE_NAME_2,
+        input_config={"gcs_source": {"uri": EXPORT_GCS_URL}},
+        project_id=GCP_PROJECT_ID,
+    )
+    # [END howto_operator_import_instance]
+
+    # [START howto_operator_delete_instance]
+    delete_instance = CloudMemorystoreDeleteInstanceOperator(
+        task_id="delete-instance", location="europe-north1", instance=INSTANCE_NAME, project_id=GCP_PROJECT_ID
+    )
+    # [END howto_operator_delete_instance]
+
+    delete_instance_2 = CloudMemorystoreDeleteInstanceOperator(
+        task_id="delete-instance-2",
+        location="europe-north1",
+        instance=INSTANCE_NAME_2,
+        project_id=GCP_PROJECT_ID,
+    )
+
+    create_instance >> get_instance >> get_instance_result
+    create_instance >> update_instance
+    create_instance >> create_instance_result
+    create_instance >> export_instance
+    create_instance_2 >> import_instance
+    create_instance >> list_instances >> list_instances_result
+    get_instance >> set_acl_permission >> export_instance
+    export_instance >> import_instance
+    export_instance >> delete_instance
+    import_instance >> delete_instance_2
+    create_instance_2 >> failover_instance
+    failover_instance >> delete_instance_2
```

Comment:
That's a crazy DAG :D

Reply:
![Screenshot 2019-08-29 at 15 50 34](https://user-images.githubusercontent.com/12058428/63981349-0ac8b200-cabf-11e9-84ea-a7601b27ccd7.png)